app/components/forum/forum-post-new.js

```diff
@@ -25,6 +25,6 @@ export default class ForumPostNewComponent extends Component {
 
   @action
   cancel() {
-    this.content = '';
+    this.set('content', '');
```

Comment:
We have to consider that this is not "Octane style" and will give a lint error once https://github.com/csvalpha/amber-ui/blob/fca981602f54d818300d07c4b346b347b6c34f1d/.eslintrc.js#L30 is reenabled.